luma/core/framebuffer.py

```diff
@@ -89,33 +105,15 @@ class full_frame(object):
     :param device: The target device, used to determine the bounding box.
     :type device: luma.core.device.device
```

Comment:
this device type probably needs `:py:class:` with backticks

Reply:
device is no longer used, so removed